src/main.cpp

```diff
@@ -15,11 +15,26 @@
 #include "logger/logger.hpp"
 #include "version.hpp"
 
+#if defined(CONFIG_BOARD_RCAR_SPIDER) || defined(CONFIG_BOARD_RCAR_SALVATOR_XS_M3_2X4G)
```

Comment:
(note) you can use `CONFIG_SOC_SERIES_RCAR_GEN4` and `CONFIG_SOC_SERIES_RCAR_GEN3` instead
if we want to add other boards in the future